HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1099,8 +1106,15 @@ def self.apply_skylights(hpxml_bldg)
       end
       if skylight.gas_fill.nil?
         if skylight.glass_layers == HPXML::WindowLayersDoublePane
-          skylight.gas_fill = HPXML::WindowGasAir
-          skylight.gas_fill_isdefaulted = true
+          if [HPXML::WindowGlassTypeLowE,
+              HPXML::WindowGlassTypeLowEHighSolarGain,
+              HPXML::WindowGlassTypeLowELowSolarGain].include? skylight.glass_type
+            skylight.gas_fill = HPXML::WindowGasArgon
```

Comment:
It looks like that if user specifies a `double-pane`, `low-e, low-solar-gain` with Aluminum frame, it fill will cause a break because we default gas fill here and there's no mapping. 

However I am not sure if we can do better. Double pane insulating low-e with metal frames seem to work better with gas fill default but double pane solar control low-e with metal frames works better with air fill default.

Reply:
That's a good point. But yeah, I think we need to just live with this limitation. The issue is really that the HEScore table doesn't include double pane solar control low-e with metal frames, but I don't want to go down the rabbit hole of trying to expand the table right now.

We publish the table in our docs and also allow users to provide their own U/SHGC, so they have ways to get around the issue should they hit it.